packages/fxa-settings/src/components/ThirdPartyAuth/index.tsx

```diff
@@ -231,8 +235,11 @@ const ThirdPartySignInForm = ({
       {!isDeeplinking ? (
         <button
           type="submit"
-          className="w-full px-2 mt-2 justify-center text-black bg-transparent border-grey-300 border hover:border-black rounded-lg text-md text-center inline-flex items-center focus-visible-default outline-offset-2"
+          className={`w-[60px] h-[60px] p-4 flex items-center justify-center rounded-full focus-visible-default outline-offset-2 ${
```

Comment:
I'd recommend adding `border border-transparent` for better support in Windows HCM (high-contrast mode for accessibility). Here's how it currently looks: 
<img width="1839" height="1077" alt="image" src="https://github.com/user-attachments/assets/1699cff7-6da9-49dc-a551-5b968b783d20" />

And how it would look after the modification:
<img width="1218" height="774" alt="image" src="https://github.com/user-attachments/assets/430d1dce-2015-4cac-a8f5-0f9183a9b993" />

The yellow outline here marks these elements as interactive.